executable/microsoft_pe.ksy

```diff
@@ -429,10 +429,8 @@ types:
       - id: time_date_stamp
         type: u4
         -orig-id: 'TimeDateStamp'
-      - id: major_version
-        type: u2
-      - id: minor_version
-        type: u2
+      - id: version
+        type: resource_directory_table_version
```

Comment:
why not to use just `version_u2` and replace everywhere we find it?

Reply:
Sure, done. I prefer to do the replacing elsewhere in the file in another PR.